File: app/src/main/java/org/uns/mtqmnuns/data/entity/Surah.kt
package org.uns.mtqmnuns.data.entity

import androidx.room.*

@Entity(tableName = "surah")
data class Surah (
    @PrimaryKey 
    @ColumnInfo(name = "id")        
    val id: Int,

    @ColumnInfo(name = "name")        
    val name: String,

    @ColumnInfo(name = "nameLatin")   
    val nameLatin: String,

    @ColumnInfo(name = "nameIndo")    
    val nameIndo: String,

    @ColumnInfo(name = "description") 
    val description: String,

    @ColumnInfo(name = "totalAyah")   
    val totalAyah: Int,

    @ColumnInfo(name = "place")       
    val place: String,
)

File: lib/data/entity/surah.dart
import 'package:drift/drift.dart';

class Surah extends Table {
  IntColumn get id => integer()();
  TextColumn get name => text()();
  TextColumn get nameLatin => text()();
  TextColumn get nameIndo => text()();
  TextColumn get description => text()();
  IntColumn get totalAyah => integer()();
  TextColumn get place => text()();

  @override
  Set<Column<Object>> get primaryKey => {id};
}
